### PR TITLE
Added K8s Example

### PR DESCRIPTION
**Pull request by rhjensen79**
Great job you have done. Just what i was looking for.
I was in the process of creating this myself, but since yours was a really great working example, i added a k8s example to it as well + some minor fixes.

### CODE REVIEW

**Review by lpasselin — APPROVED** [review]

**Comment by lpasselin** [comment]
wow thanks a lot for the PR! I really wasn't expecting this!
Thanks for the minor corrections and the k8 example.

If you have some time, can you add a script to launch the k8 example? If you prefer, we can merge right away.
I will do the same in a future PR for the docker-compose examples.

File: README.md
Comment (lpasselin) [inline README.md]:
```suggestion
Build and tag the image (or push to a registry). Change `YOUR_IMAGE` in the deployment.yaml file.
```

**Comment by rhjensen79** [comment]
Np at all.
I wanted to create it myself, but your repo and code, looked so nice, that it was a pleasure to add to that.

Happy to do some automation around it. 
I will also some actions, with auto build of images, so they are easy to consume. 

Expect to see a couple more PR :-)

File: README.md
Comment (rhjensen79) [reply README.md]: Agree on that change

**Comment by rhjensen79** [comment]
Sorry closed it by mistake. 

Let me know if you need more in the PR. I will create another for the other changes.

**Comment by rhjensen79** [comment]
Added a bit of automation, and updated the documentation.
Deployment don't work, until you approve the github action, and run it, so a new container image is created in your repo.

**Comment by lpasselin** [comment]
I don't have a lot of experience with github actions but would like to experiment first to make sure everything works and is clean. Let's merge in a new dev branch to play with the github actions. I'll add you as maintainer of the repo to simplify things.

**Comment by rhjensen79** [comment]
Deal. I will update the action to work on the dev branch as well, and only trigger when changes are made in the k8s folder.

**Pull request merged**
